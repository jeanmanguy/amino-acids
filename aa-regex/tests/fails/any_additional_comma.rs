use aa_regex::*;

fn main() {
    let wrong = any!('C',);
    let wrong = any!(,);
}
