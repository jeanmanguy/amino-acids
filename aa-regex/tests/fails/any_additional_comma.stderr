error: unexpected end of input, expected literal
 --> $DIR/any_additional_comma.rs:4:17
  |
4 |     let wrong = any!('C',);
  |                 ^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected literal
 --> $DIR/any_additional_comma.rs:5:22
  |
5 |     let wrong = any!(,);
  |                      ^
